lib/exchange_clients/bitfinex/index.js

```diff
@@ -1,8 +1,6 @@
 'use strict'
 
-const Promise = require('bluebird')
```

Comment:
not sure but seems we can remove bluebird also since it doesn't seem to utilize it's features

Reply:
its used in another place in the codebase still

Reply:
would handle "remove bluebird" as separate task